rds.tf

```diff
@@ -0,0 +1,42 @@
+data "aws_iam_policy_document" "rds_for_github" {
+
+  statement {
+    sid    = "AllowListDescribe"
+    effect = "Allow"
+    actions = [
+      "rds:Describe*",
+      "rds:ListTagsForResource",
+    ]
+    resources = ["*"]
+  }
+
+  statement {
+    sid    = "AllowGetOwn"
+    effect = "Allow"
+    actions = [
+      "rds:CreateDBSnapshot",
+      "rds:ModifyDBInstance",
+      "rds:RebootDBInstance"
```

Comment:
Should we add `rds:DownloadCompleteDBLogFile` and `rds:DownloadDBLogFilePortion`? Would this be useful for users?

Reply:
added